src/main/c/trufflemock/trufflemock.c

```diff
@@ -0,0 +1,309 @@
+/*
```

Comment:
@eregon I've added a generate tool script and updated the `trufflemock.c` file to its output.